contracts/v2/PolygonZkEVMBridgeV2.sol

```diff
@@ -440,7 +440,7 @@ contract PolygonZkEVMBridgeV2 is
         bytes calldata metadata
     ) external payable ifNotEmergencyState {
         // If exist a gas token, only allow call this function without value
-        if (msg.value != 0 && address(WETHToken) != address(0)) {
+        if (msg.value != 0 && gasTokenAddress != address(0)) {
```

Comment:
Latest conversations around outpost may add the possibility to identify native gasToken as:
- `gasTokenNetwork: address(0)`
- `gasTokenNetwork: X`

Therefore, the check is incomplete. Althought it is not yet supported in the PP I would identify ETH as:
- `gasTokenNetwork != address(0) && gasTokenNetwork != 0`